website/content/en/docs/dev/internals/_index.md

```diff
@@ -37,6 +37,7 @@ Metadata:
 - `protected`: empty file, used by `limactl protect`
 
 cloud-init:
+- `cloud-config.yaml`: cloud-init configuration, for reference only.
```

Comment:
Factory-reset should remove this yaml?

Reply:
It was created with the instance (`create`)

Maybe we should make sure to update it, like if the global config has changed.
But since we don't update lima.yaml either, we could also just leave it as-is...

Reply:
Shouldn't it be (re-)created on start?

Reply:
It is updated on start now, d22b200a33f1b4ad49ddd1f7078deedcac7702ce

Reply:
The main usage of this file was like:
1) `limactl create` the instance, using the "ext" driver
2) use the `cloud-config.yaml` to create an external VM
3) `limactl start` the instance, connect to existing VM

So it needs to be created before the start command runs.

Reply:
What about factory-reset?

Reply:
I'm not entirely sure what it _means_ to "factory reset" an instance, and it's not actually documented in the command... But usually it just means to restore settings to the defaults that it arrived with. But we don't reset the lima.yaml with the template, so I did not reset the generated cloud-config either. But we could add a line to regenerate it, maybe it is useful if you change the global '_config` or something? (Like loadDotSSHPubKeys)

Reply:
Yes, it should be either removed or regenerated